oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -0,0 +1,250 @@
+Deploy OKE Nodes using Ubuntu Images
+====================================
+
+Ubuntu images are available for worker nodes on Oracle Kubernetes Engine (OKE) in Oracle Cloud. Currently there are only a select number of suites and Kubernetes versions supported due to this being a Limited Availability release. 
+
+Available Releases
+------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Ubuntu Release
+     - OKE Version
+     - Location
+   * - 22.04 (Jammy Jellyfish)
+     - 1.29
+     - `Link <https://intcanonical.objectstorage.us-phoenix-1.oci.customer-oci.com/p/vpqQtYASl8IooEZ_sxfKDnzUkF1b-3lQmXPC_rXf4zARQYoW7ncE8BGGxNqdUuGa/n/intcanonical/b/oke-shared/o/>`_
+
+
+Networking Plugin Availability
+------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Node Type
+     - Plugin
+     - Supported
+   * - Managed
+     - Flannel
+     - Yes
+   * - 
+     - VCN Native
+     - Yes
+   * - Self-Managed
+     - Flannel
+     - Yes
+   * - 
+     - VCN Native
+     - No
+
+
+Prerequisites
+-------------
+
+You'll need:
+
+- Oracle Cloud compartment to create the nodes.
+
+- Configured and running OKE cluster on Oracle Cloud.
+
+- Oracle's ``oci`` CLI installed.
+
+- ``kubectl`` installed (Self-Managed only).
+
+
+Download an Ubuntu image
+------------------------
+
+
+To download an image, select a version from the `available releases <#available-releases>`_ and make note of the image path for the image you choose. The image path conforms to the following format:
```

Comment:
I think we need a section on how to interpret the JSON listing provided by #available-release links.  It is a sorted array where the most recent image will be at the end (with the last two entries being the latest image and the latest image manifest respectively).

I'd also like to add a CLI snippet, along the lines of 
```
 curl https://intcanonical.objectstorage.us-phoenix-1.oci.customer-oci.com/p/vpqQtYASl8IooEZ_sxfKDnzUkF1b-3lQmXPC_rXf4zARQYoW7ncE8BGGxNqdUuGa/n/intcanonical/b/oke-shared/o/ | jq ".[][-2] | .name"
```
to automate getting the latest.

Design note (somewhat off topic here):
We could also create a `<suite>/oke-<version>/current` prefix in the bucket, that contains the latest image released, which might be easier to navigate/inspect.